### DISCUSSION TITLE
💻Server load distribution! and reduction upto 50% load on server. ❗

### DISCUSSION THREAD

**Discussion by Chintan-19** [Pavankumar-Mhaske/Ecommerce-Business-App#3] [Q&A] [answered]
👋Hey, Pavan can you explain how did you make the functionality of the file uploading and how did you distributed the server load?

**Comment by Pavankumar-Mhaske** [comment, marked as answer]
👋Hi chintan, Actually i used the cloudinary 📅 services for the file management stuff which is helping me to avoid ❌ storing images data on the database... ✅
Process 💯 	🚀Implemented a load distribution system in the application, reducing server load by up to 50%. The process involves compressing and creating a temporary instance of images uploaded by users on the server. Simultaneously, the compressed image is sent to Cloudinary's cloud storage, where it is stored and assigned public keys, public IDs, and access IDs. These IDs are then stored in the database associated with the user's schema. Once completed, the user is shown the image using the Cloudinary IDs, and the temporary image on the server is deleted. 💖

**Reply by Chintan-19** [reply]
Ohh! that's nice, Thanks pavan.
